jjb/dynamic/scale-ci_install_osp.yml

```diff
@@ -23,7 +23,7 @@
         do
         
         # Get the day of week and set networkType as OpenshiftSDN on odd days and kuryr on even days in case no network is defined manually.
-        if [ $OPENSHIFT_NETWORK_TYPE == "" ]
+        if [ "$OPENSHIFT_NETWORK_TYPE" == "" ]
```

Comment:
Maybe use -z flag for the check? Thoughts?

Reply:
totally agree, done!